product/stream_processor/integrations/events/models/input.py

```diff
@@ -0,0 +1,21 @@
+from datetime import datetime
+from typing import Generic, TypeVar
+
+from pydantic import BaseModel, ConfigDict, Field
+
+AnyModel = TypeVar('AnyModel', bound=BaseModel)
+
+
+class EventMetadata(BaseModel):
+    event_name: str
```

Comment:
let's use the same Field with min length definition i used so it's non empty etc.